migrations/sql/V1.29.1__alter_mine_party_appt_type_code.sql

```diff
@@ -0,0 +1,46 @@
+
+ALTER TABLE public.mine_party_appt_type_code add person boolean NOT NULL DEFAULT 'false' ;
```

Comment:
Typically in code tables we wouldn't have additional columns, but I am curious about these ones... can we walk through it? The Person and Organization columns make me think we may need to model them a little differently as these look like potential code values in themselves?

Reply:
Thanks. I am just taking a look at the model to see if there are any opportunities to use the Party Type Code to our advantage... maybe we can chat about that?

What is the "grouping level" intended for? What would Group 1 mean vs. Group 2 and Group 3?

Reply:
Agreed. The person bool is essentially the party_type_code. Person bool also isn't flexible enough because there could be 3 states (person, company, person & company). I think an xref table between mine_party_appt_type_code and party_type_code is a good solution. I also might let this go for this PR and refactor it later.

Reply:
@itchison @klefler An xref table would be good, but I'd like to define a group type / name along with it. "Key Contacts", "Specialists". The intent is to refactor.

Reply:
Agreed as well. I'm happy to keep as-is for this PR and refactor later.

I'm also looking at the grouping_level... as seen as the demo today, these 3 categories have specific meaning. Typically, we would want to have these categories as lookup/code values as well as it will make things easier to maintain. It will also help to bring more understanding and business meaning to the values... for example; I didn't know grouping_level = 3 means "key contacts" versus 2 = "Specialists", etc.

Again, happy to leave it for this PR but we might want to put some thought into this for a later refactor.

Reply:
I think we're on the same page on this. I will refactor to an xref table, and create a group_code table in a separate PR